### DISCUSSION TITLE
멘토링 issue 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#65] [General]
멘토링 내용을 정리하고 추적을 조금 하고 싶어서 issue로 관리하려고 하는데 멘토님을 Assignees에 걸어도 되나요..?

지금은 임시로 작성했지만 과거 멘토링이나 내용들을 정리할 생각입니다..!

생각중인 방법은 멘토링이 끝나면 다음 멘토링 날짜와 시간을 적어서 이슈로 발행하고 종료가 되면 멘토링 내역을 적고(질문 답변)마감하려고 합니다...!

**Comment by fkdl0048** [comment]
++ 지금까지 받은 질문, 답변 정리해도 될까요?

**Reply by jongfeel** [reply]
네 좋습니다. 오히려 그렇게 해주시면 더 좋죠.
저도 비슷한 방법으로 제 github에 정리하고 있었습니다. - 실제 미팅 내용은 정안님 repository에 적었습니다.
